hack/lib/util.sh

```diff
@@ -136,14 +136,21 @@ kube::util::gen-doc() {
   # create the list of generated files
   ls "${tmpdir}" | LC_ALL=C sort > "${tmpdir}/.files_generated"
 
-  # remove all old generated file from the destination
   while read file; do
+    # Add analytics link to generated .md files
+    if [[ "${file}" == *".md"* ]]; then
+      local link destroot
+      destroot=$(dirname ${dest})
+      link=$( kube::util::analytics-link "${dest#$destroot/}${file}" )
```

Comment:
nitpick: curly braces: `"${dest#${destroot}/}${file}"`

Also, if `${destroot}` is `$(dirname "${dest}")`, then isn't `${dest#${destroot}/}` the same as `$(basename "${dest}")`?

Usually I like to add an explicit "/" between the directory and the file name, so maybe:

```shell
  local path link
  path=$(basename "${dest}")/${file}
  link=$(kube::util::analytics-link "${path}")
```